compiler/qsc_frontend/src/lower/tests.rs

```diff
@@ -239,9 +239,9 @@ fn lift_local_newtype() {
                         ctl-adj: <none>
                 Item 2 [49-67] (Internal):
                     Parent: 1
-                    Type (Ident 6 [57-60] "Bar"): Udt:
-                        base: Int
-                        fields:"#]],
+                    Type (Ident 6 [57-60] "Bar"): UDT [49-67]:
```

Comment:
Now that there is some more logic to ensure spans and names show up for UDTs, this file should have a few more test cases with more complex UDT structures (some with names, some without, nested tuples, etc) to make sure the corresponding lowered HIR matches the expected form.

Reply:
I've added a couple of tests, as suggested.